pages/_app.js

```diff
@@ -1,5 +1,8 @@
 import GlobalStyle from "@/styles";
 import Head from "next/head";
+import { SWRConfig } from "swr";
+
+const fetcher = (url) => fetch(url).then((response) => response.json);
```

Comment:
```suggestion
const fetcher = (url) => fetch(url).then((response) => response.json());
```
Don't forget to call json :)

Reply:
Oh, I completely forgot to add the environment variable. I've fixed it!